pkg/config/utils/tags.go

```diff
@@ -5,7 +5,34 @@
 
 package utils
 
-import "github.com/DataDog/datadog-agent/pkg/config"
+import (
+	"sort"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// removeDuplicatesAndSort returns a new array, sorted without duplicates.
+func removeDuplicatesAndSort(elements []string) []string {
+	// isolate unique elements
+	found := make(map[string]bool)
+	unique := []string{}
+
+	for v := range elements {
+		if !found[elements[v]] {
+			unique = append(unique, elements[v])
+			found[elements[v]] = true
+		}
+	}
+
+	// sort the array
+	sort.Strings(unique)
+
+	// copying the array with exactly enough capacity should make it more resilient
+	// against cases where `append` mutates the original array
+	res := make([]string, len(unique))
+	copy(res, unique)
+	return res
+}
```

Comment:
This functions looks copied/pasted from `pkg/util/sort_uniq.go`.
Could we refactor the code to avoid copy/paste and have only a single definition of this function ?

Reply:
There was a circular dependency between the config package and the util package. I think it the solution would be to find another package where we could put this function.